AlphaWallet/TokenScriptClient/Models/XMLHandler.swift

```diff
@@ -932,11 +932,11 @@ extension XMLHandler {
     }
 
     fileprivate static func getTokenScriptTokenInstanceCardElements(fromRoot root: XMLDocument, xmlContext: XmlContext) -> XPathObject {
-        return root.xpath("/token/cards/card".addToXPath(namespacePrefix: xmlContext.namespacePrefix), namespaces: xmlContext.namespaces)
+        return root.xpath("/card[@type=\"token\"]".addToXPath(namespacePrefix: xmlContext.namespacePrefix), namespaces: xmlContext.namespaces)
```

Comment:
FYI. You can use single quotes to avoid escaping.

Reply:
Does it actually work? "/card" means the root element is `/card` which is true only for X-Dai bridge.